theme/styles/global.scss

```diff
@@ -112,7 +112,7 @@
         grid-column: 1/3;
         grid-row: 1/3;
 
-        img {
+        .ProductGridItem__image {
           min-height: 31.25rem!important;
```

Comment:
Does this need the `!important` declaration?

Reply:
seems ok without!